backend/routes/programs.js

```diff
@@ -39,3 +44,123 @@ router.post("/", function(req, res) {
 });
 
 module.exports = router;
+
+router.get("/:programId/users", function(req, res) {
+  db["Authentication"].users
+    .find({
+      programs: {
+        $elemMatch: {
+          id: req.params.programId
+        }
+      }
+    },
+    {
+      userId: 1,
+      email: 1,
+      name: 1,
+    })
+    .then(function(found) {
+      res.json(found);
+    })
+    .catch(function(err) {
+      res.send(err);
+    });
+});
+
+router.post("/:programId/users", async function(req, res) {
```

Comment:
I can't decide if this should go in `user.js` or `programs.js`... programs makes more sense for the client code, but most of the logic here is on the user.

Reply:
I think it makes sense here considering the endpoint uses a programId. I don't think we plan to use create user anywhere else but we could export the create user part from user.js